galter_subjects_utils/reader.py

```diff
@@ -56,6 +58,10 @@ def __iter__(self):
         """Iterate over terms."""
         yield from MeSH.load(self._filepath, self._filter)
 
+    def read(self):
```

Comment:
For this one and other "yielders" I will investigate a slightly better pattern (for future PR), since as it is, they can only be iterated once for a given Reader objects.